packages/application-config/src/validate-config.ts

```diff
@@ -10,7 +10,7 @@ type ErrorAdditionalProperty = ErrorObject<
 >;
 type ErrorEnum = ErrorObject<'enum', { allowedValues: string[] }>;
 
-const ajv = new Ajv({ strict: true });
+const ajv = new Ajv({ strict: true, useDefaults: true });
```

Comment:
This allows the schema validator to inject default values to the config, according to the schema.